packages/vaadin-lumo-styles/src/components/date-picker-overlay.css

```diff
@@ -7,20 +7,8 @@
   [part='overlay'] {
     display: flex;
     flex: auto;
-
-    /*
-  Width:
-      date cell widths
-    + month calendar side padding
-    + year scroller width
-  */
-    /* prettier-ignore */
-    width:
-  calc(
-      var(--lumo-size-m) * 7
-    + var(--lumo-space-xs) * 2
-    + 57px
-  );
+    /* Width: date cell widths + month calendar side padding + year scroller width */
+    width: calc(var(--lumo-size-m) * 7 + var(--lumo-space-xs) * 2 + 57px);
```

Comment:
IMO this makes more sense in a single line, both the code and the comment look readable enough.